<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountDetail</class>
 <widget class="QWidget" name="accountDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar{
	border-radius: 5px;
     border: none;
     text-align: center;
     background: rgb(192,192,192);
}
QProgressBar::chunk {
	border-radius: 5px;
     border: none;
     background: rgb(59,66,85);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="layout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QWidget" name="xumi_widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" rowspan="3">
            <widget class="QLabel" name="xumi_icon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="portraitUI.qrc">:/world/world/UI_ChapterInnerIcon_Xumi.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="xumi_DreamTree">
             <property name="text">
              <string>梦之树：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="xumi_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="xumi_reputation">
             <property name="text">
              <string>声望等级：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QProgressBar" name="xumi_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>须弥</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="enkanomiya_widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0" rowspan="3">
            <widget class="QLabel" name="enkanomiya_icon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="portraitUI.qrc">:/world/world/UI_ChapterInnerIcon_Enkanomiya.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="xumi_DreamTree_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="enkanomiya_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="xumi_reputation_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QProgressBar" name="enkanomiya_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>渊下宫</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="cengyan_widget" native="true">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="3">
            <widget class="QLabel" name="cengyan_Lumenstone">
             <property name="text">
              <string>流明石触媒等级：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QProgressBar" name="cengyan_under_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>层岩巨渊</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="cengyan_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="cengyan_under_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="4">
            <widget class="QLabel" name="cengyan_icon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="portraitUI.qrc">:/world/world/UI_ChapterInnerIcon_ChasmsMaw.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QProgressBar" name="cengyan_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="daoqi_widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_22">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_23">
           <item row="0" column="0" rowspan="3">
            <widget class="QLabel" name="daoqi_icon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="portraitUI.qrc">:/world/world/UI_ChapterInnerIcon_Daoqi.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="daoqi_Oraionokami">
             <property name="text">
              <string>神樱眷顾：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="daoqi_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="daoqi_reputation">
             <property name="text">
              <string>声望等级：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QProgressBar" name="daoqi_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>稻妻</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QWidget" name="liyue_widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_24">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_25">
           <item row="1" column="1">
            <widget class="QLabel" name="liyue_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="3">
            <widget class="QLabel" name="liyue_icon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="portraitUI.qrc">:/world/world/UI_ChapterInnerIcon_Liyue.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="liyue_reputation">
             <property name="text">
              <string>声望等级：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="xumi_DreamTree_6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QProgressBar" name="liyue_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>璃月</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QWidget" name="dragonspine_widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_26">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_27">
           <item row="0" column="0" rowspan="3">
            <widget class="QLabel" name="dragonspine_icon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="portraitUI.qrc">:/world/world/UI_ChapterInnerIcon_Dragonspine.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="dragonspine_Eldertree">
             <property name="text">
              <string>忍冬之树：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="dragonspine_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="xumi_reputation_6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QProgressBar" name="dragonspine_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>龙脊雪山</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QWidget" name="mengde_widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_28">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_29">
           <item row="1" column="1">
            <widget class="QLabel" name="mengde_exploration">
             <property name="text">
              <string>探索度：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="3">
            <widget class="QLabel" name="mengde_icon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="portraitUI.qrc">:/world/world/UI_ChapterInnerIcon_Mengde.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="mengde_reputation">
             <property name="text">
              <string>声望等级：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="xumi_DreamTree_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QProgressBar" name="mengde_progressBar">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>蒙德</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="portraitUI.qrc"/>
 </resources>
 <connections/>
</ui>
